crates/storage/src/lib.rs

```diff
@@ -17,12 +19,21 @@ table!(
     /// Block bodies table.
     ( Bodies ) BlockNumber => BlockBodyRLP
 );
+table!(
+    /// Account infos table.
+    ( AccountInfos ) AddressRLP => AccountInfoRLP
```

Comment:
Can we use `Address` directly here?

Reply:
We need it to implement `Encodable` in order to be used in the table, and we cannot implement it as it comes from the an external crate